client.go

```diff
@@ -117,6 +119,71 @@ func (c *client) GetBug(id int) (*Bug, error) {
 	return bugs[0], nil
 }
 
+// GetBugComments retrieves the comments of a Bug from the server
+// https://bugzilla.readthedocs.io/en/latest/api/core/v1/comment.html#get-comments
+func (c *client) GetBugComments(id int, values *url.Values) ([]Comment, error) {
+	logger := c.logger.WithFields(logrus.Fields{methodField: "GetBugComments", "id": id})
+	url := fmt.Sprintf("%s/rest/bug/%d/comment", c.endpoint, id)
+
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		return nil, err
+	}
+	if values != nil {
+		req.URL.RawQuery = values.Encode()
+	}
+	raw, err := c.request(req, logger)
+	if err != nil {
+		return nil, err
+	}
+	var parsedResponse struct {
+		Bugs map[string]*struct {
+			Comments []Comment `json:"comments,omitempty"`
+		} `json:"bugs,omitempty"`
+	}
+	if err := json.Unmarshal(raw, &parsedResponse); err != nil {
+		return nil, fmt.Errorf("could not unmarshal response body: %v", err)
+	}
+	if len(parsedResponse.Bugs) != 1 {
+		return nil, fmt.Errorf("did not get one bug, but %d: %v", len(parsedResponse.Bugs), parsedResponse.Bugs)
+	}
+	for _, comments := range parsedResponse.Bugs {
+		return comments.Comments, nil
+	}
+	return nil, nil
+}
+
+// GetBugHistory retrieves the history of a Bug from the server
+// https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#bug-history
+func (c *client) GetBugHistory(id int, values *url.Values) ([]History, error) {
```

Comment:
Same issue here, probably should be a real golang struct which we turn into Values.

Reply:
fixed (without option structs)